service.go

```diff
@@ -180,3 +185,29 @@ func (svc *Service) createResponse(kind int, initialEvent *nostr.Event, content
 	}
 	return resp, nil
 }
+
+func (svc *Service) hasPermission(app *App, event *nostr.Event, paymentRequest *decodepay.Bolt11) bool {
+	appPermission := AppPermission{}
+	findPermissionResult := svc.db.Find(&appPermission, &AppPermission{
+		AppId:     app.ID,
+		NostrKind: event.Kind,
+	})
+	if findPermissionResult.RowsAffected == 0 {
+		return false
```

Comment:
Do we need to migrate all existing apps to have a permission if we return false here?

Reply:
moved question to other PR